core-rust/mesh-api-server/src/mesh_api/types.rs

```diff
@@ -0,0 +1,35 @@
+use crate::engine_prelude::*;
+use crate::prelude::*;
+
+#[derive(Debug, Clone, EnumIter, Display, EnumString)]
+pub(crate) enum MeshApiOperationTypes {
```

Comment:
This should probably be called `MeshApiOperationType` (it's typical to avoid `s` on enums)

Reply:
Done